src/hotspot/share/memory/memoryReserver.cpp

```diff
@@ -450,6 +450,9 @@ ReservedSpace HeapReserver::Instance::try_reserve_range(char *highest_start,
 
       release(reserved);
     }
+
+    if (p2u(attach_point) <= stepsize)
+      break;  // Make sanizier silent and avoid warp around.
```

Comment:
Sanitizer isn't interesting.  The point is this avoids pointer underflow.  Also, braces are required for multiline
`if` statements - see style guide.

Reply:
Thanks @kimbarrett, the comment has been updated, and the braces has been added.